app/views/hq/partners/index.html.erb

```diff
@@ -1,13 +1,17 @@
-<div class="action_items">
-  <div class="class">
-    <%= link_to('New Partner', new_hq_partner_path,
-                class: 'btn btn-default', role: 'button') %>
+<div class="panel-heading">
+  <div class="action_items">
+    <div class="class">
```

Comment:
should you also remove this `class`here, since you are editing this part anyway?